### PR TITLE
Remove "log_stderr" in favor of always logging errors to the log panel

### PR DESCRIPTION
**Pull request by predragnikolic**
Remove `log_stderr` in favor of always logging errors to the log panel.

### CODE REVIEW

File: docs/src/guides/client_configuration.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Here is an example of the `LSP.sublime-settings` file with configurations for th
 ```js
 {
   // General settings
-  "log_stderr": true,
+  "log_debug": true,
```
Comment (rchl) [inline docs/src/guides/client_configuration.md:15]: Should we suggest this one really?

Comment (predragnikolic) [reply docs/src/guides/client_configuration.md:15]: I could put `show_view_status` instead of it, or some other setting?

Comment (rwols) [reply docs/src/guides/client_configuration.md:15]: `"show_diagnostics_panel_on_save": 0` is IMO a good example

Comment (predragnikolic) [reply docs/src/guides/client_configuration.md]: b1fefb1

**Review by rwols — APPROVED** [review]

**Pull request merged**